### DISCUSSION TITLE
scripts down??

### DISCUSSION THREAD

**Discussion by ForTheRush** [community-scripts/ProxmoxVE#542] [General]
looks like theres not available scripts on the site anymore. tried phone and computer and it just keeps showing me this blank page with zero available scritps to use. 

is there some sort of mass edit happening or is it just github tripping?

![no-scripts](https://github.com/user-attachments/assets/731efd34-3946-41d2-8283-9981d1199379)

**Comment by MickLesk** [comment]
Already fixed by @havardthom 

Was my mistake (forgetten comma in json File)
